Multisig/UI/ENS/EnterENSNameView.swift

```diff
@@ -43,6 +43,7 @@ struct EnterENSNameView: View {
             self.presentationMode.wrappedValue.dismiss()
             self.onConfirm(self.model.address!.hex(eip55: true))
         }
+        .font(Font.body.bold())
```

Comment:
body.bold is used now in many areas, I prefer to create new font style for it, or use Headline which is semibold

Reply:
Ok, I'll create a tech-debt ticket to do that throughout the app.